.drone.yml

```diff
@@ -230,3 +230,34 @@ steps:
       VERSION: "${DRONE_TAG}"
     depends_on:
       - clone
+
+---
+kind: pipeline
+type: kubernetes
+name: migration deploy tag
+trigger:
+  event:
+    - tag
+  branch:
+    - main
+depends_on:
+  - build tag
+
+steps:
+  - name: deploy migrator to cs-dev
+    image: quay.io/ukhomeofficedigital/kd
+    commands:
+      - cd kube
+      - ./deploy.sh
+    environment:
+      ENVIRONMENT: cs-dev
+      DEPLOYMENT_TYPE: migrator
+      SQS_SECRET_NAME: case-migrator-sqs
+      CASE_CREATOR_PORT: 10943
```

Comment:
Why not default to 8080 like every other service? You should template the service so you can interact. Although there won't be any endpoint on this service I would have thought...

Reply:
Done.